### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mudassirsh**
Commit 04/11  

Please review. Thanks

### CODE REVIEW

**Comment by k-mughal** [comment]
content looks good to me, but i see you have branch conflict that must be resolved, plz look into merge pull requests command

**Review by k-mughal — APPROVED** [review]
i noticed some spaces issue in the content which have been fixed

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@
 - **Desktop Env:**		GUI on the top of operating system.			GNOME, KDE, Xfce and Fluxbox	are the examples of desktop environments.
 - **Command line:**		Interface for typing commands on top of the operating system.
 - **Shell:**			Command line interpreter that interprets the command line input and instructs the operating system to perform any necessary tasks and command. i.e. bash, tcsh, and zsh
-			            Large organizations use commercially supported Distributions from RedHad, Susie and Canonical (Ubuntu) 
+			        Large organizations use commercially supported Distributions from RedHad, Susie and Canonical (Ubuntu) 
                   Linux is a full multitasking multiuser operating system with built-in networking and service processes known as deamons.
```
Comment (mudassir-sh) [inline README.md:37]: Remove Extra spacing.

**Review by mudassir-sh — CHANGES_REQUESTED** [review]

**Comment by mudassirsh** [comment]
All reviewed.

**Pull request merged**